target/linux/ath79/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -87,6 +87,10 @@ board=$(board_name)
 case "$FIRMWARE" in
 "ath10k/cal-pci-0000:00:00.0.bin")
 	case $board in
+	comfast,cf-wr650ac-v1|\
+	comfast,cf-wr650ac-v2)
```

Comment:
You can include yuncore,a770 from below here ...

Reply:
done